main/acle.md

```diff
@@ -2837,14 +2838,11 @@ the selection algorithm is platform dependent, where with platform means
 CPU/Vendor/OS as in the target triplet.
 2. The selection is permanent for the
 lifetime of the process.
-3. Only those versions could be considered where all
-dependent features are available.
-
-Rules of version selection are in order:
-
-4. Select the most specific version else
-5. select the version with the highest priority else
-6. `"default"` is selected if no other versions are suitable.
+3. Among any two versions, the higher priority version is determined by
+identifying the highest priority feature that is specified in exactly one of
+the versions, and selecting that version.
```

Comment:
This wording makes clear what happens when choosing between "bf16" and "bf16, sme".
It is unclear to me what happens when choosing between "sme" and "sme, bf16".

Reply:
Actually it's clear after re-reading this.